### PR TITLE
Add section about video player compatibility

### PR DESCRIPTION
**Pull request by papr**
And add ffmpeg command for comaptible transcoding.

### CODE REVIEW

File: src/developer/core/recording-format.md [hunk 1]
```diff
@@ -62,6 +62,14 @@ And their corresponding timestamp files should follow the pattern:
 
 - `world_timestamps.npy`, `world_001_timestamps.npy`, `world_002_timestamps.npy`, ...
 
+### Video Player Compatibility
+
+In order to open the video files with the majority of video players, you will have to
+**transcode the video stream encoding from mjpeg to h264** and **change the color format
+from yuvj422p to yuv420p**. You can use the following [ffmpeg](https://ffmpeg.org/)
```
Comment (willpatera) [inline src/developer/core/recording-format.md:69]: I would recommend adding title attr to the link to ffmpeg for accessibility purposes. 

```suggestion
from yuvj422p to yuv420p**. You can use the following [ffmpeg](https://ffmpeg.org/ "FFmpeg - software solution for recording, converting, and streaming audio and video")
```

File: src/developer/core/recording-format.md [hunk 2]
```diff
@@ -62,6 +62,14 @@ And their corresponding timestamp files should follow the pattern:
...
+command to perform this transcoding:
+> `ffmpeg -i world.mp4 -vf format=yuv420p world_compatible.mp4`
```
Comment (willpatera) [inline src/developer/core/recording-format.md:71]: Do we need `>`?

**Review by willpatera — COMMENTED** [review]
please see comments/questions @papr

**Comment by willpatera** [comment]
@papr did you see my comments above?

File: src/developer/core/recording-format.md (hunk 2 above)
Comment (papr) [reply src/developer/core/recording-format.md:71]: I wanted to be sure that it is being rendered into a newline. Not sure if `>` is required for that. I will give it a try locally.

Comment (papr) [reply src/developer/core/recording-format.md:71]: Without `>`
Screenshot 2020-12-09 at 11 27 25

With `>`
Screenshot 2020-12-09 at 11 27 35

@willpatera Please let me know which one you favor. I am ok with both.

Comment (willpatera) [reply src/developer/core/recording-format.md:71]: @papr use 

triple backticks to make a code block

https://github.com/pupil-labs/pupil-docs#code-blocks

Comment (papr) [reply src/developer/core/recording-format.md]: Done.

**Pull request merged**